src/Lumper.UI/Views/LogViewer/LogViewerView.axaml.cs

```diff
@@ -107,6 +112,14 @@ private void AddLogMessage(LogMessage logMessage)
                 );
 
                 Logs.Inlines.Add(new Run(ex.Message) { Foreground = Brushes.IndianRed });
+                if (ex.StackTrace is not null)
+                {
+                    Logs.Inlines.Add(new LineBreak());
+                    string indent = new(' ', 8 + 26);
```

Comment:
The `8 + 26` seems arbitrary. Can you use `\t` instead?

Reply:
I'm going for very specific padding here, not keen on tabs, and this makes more sense if you check the rest of the file. I could do a named constant for `26` but honestly with immediate-mode style UI like this I find the code easier to read when constants are written explicitly like this.